images/gp-logo-green.svg

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-50 127.5 183 30" enable-background="new -50 127.5 183 30" xml:space="preserve">
+<title>gp-logo</title>
+<g>
+	<path fill="#66CC00" d="M7.7,134.1c-0.5,0.4-1.4,0.5-5.9,1.3c0,0-0.2,0-0.9,0.4c-1.1,0-2.8,0.8-3.3,1.1c-0.2,0.1-0.5,0.7-0.6,1.8

```

Comment:
Instead of using three different files, why not [style the fill via CSS](https://www.sitepoint.com/real-world-use-of-css-with-svg/)?